documentation/commands.md

```diff
@@ -1,400 +1,13 @@
 # Command Reference
 
-Although the tutorial gives a large amount of information, if you find
-remembering the commands too complicated, here's a reference. Remember
-that you can type out multiple commands in one go; for example:
-`BAW231 fh 090 d 30 sp 180` will work as well as all three commands run
-separately. Some commands have "aliases" that are shorter to type. An
-example of that would be the `takeoff` command which has an alias `to`.
+Here you can find a list of all commands available in the game, what they do, and how to use them. There are two different sets of commands: _aircraft commands_ and _scope commands_. To toggle between entering aircraft commands and scope commands, use the `tab` key.
 
-## Table of Contents
+### Aircraft Commands Overview
+As you might have guessed, aircraft commands are the commands used to issue instructions and communicate with aircraft. Many commands have "aliases" which act as abbreviations of the full command word, thus saving you time and effort. For example, `AAL123 to` will work exactly the same as `AAL123 takeoff`. You may chain as many aircraft commands in a single transmission as you like.
 
-[Departure Commands](#departure-commands)
-- [Cleared as Filed](#cleared-as-filed)
-- [Climb via SID](#climb-via-sid)
-- [Takeoff](#takeoff)
-- [Taxi](#taxi)
+[Aircraft Command Reference](aircraft-commands.md)
 
-[Arrival Commands](#arrival-commands)
-- [Descend via STAR](#descend-via-star)
-- [Land](#land)
+### Scope Commands Overview
+Scope commands are what controllers type into their radar screens for things like handoffs, moving data blocks, changing brightness settings, and similar tasks.
```

Comment:
Can you type something into to a screen?  That reads funny. 

Terminal maybe?

Reply:
Well yeah, trouble is that I'm trying to avoid using too many technical terms? Really I was cheating to silently explain that the "scope" is the "radar screen". But I agree it reads funny. I'll just go for the accurate verbage, and if you have any other suggestions I can change again.